rust/libnewsboat-ffi/src/keymap.rs

```diff
@@ -60,6 +64,24 @@ fn tokenize_operation_sequence(
     }
 }
 
-fn operation_tokens(input: &Operation) -> &Vec<String> {
-    &input.tokens
+fn tokenize_binding(input: &str, parsing_failed: &mut bool) -> ffi::Binding {
+    match libnewsboat::keymap::tokenize_binding(input) {
+        Some(binding) => {
+            let operations = binding
+                .operations
+                .into_iter()
+                .map(|tokens| ffi::Operation { tokens })
+                .collect();
+            ffi::Binding {
+                key_sequence: binding.key_sequence,
+                contexts: binding.contexts,
+                operations,
+                description: binding.description.unwrap_or("".to_owned()),
```

Comment:
Nitpick: It's better to do `unwrap_or_else(|| String::new())`, which avoids allocating a new `String` unless it's really needed. I'm surprised Clippy didn't suggest it in CI.

Reply:
It looks like `String::new()` does not allocate any buffer
From https://doc.rust-lang.org/std/string/struct.String.html#method.new:
> Given that the String is empty, this will not allocate any initial buffer.

[A clippy lint](https://github.com/rust-lang/rust-clippy/issues/2972) was added a while back to suggest `String::new()` over `"".to_string()`
However, that was later changed to a "pedantic" lint.

I will leave it as-is for now, but thanks for the suggestion! It is nice to be aware of the `_or_else()` variant of unwrapping. I will try to keep it in mind for situations where it makes sense.